huaweicloud/services/er/resource_huaweicloud_er_instance.go

```diff
@@ -364,14 +407,75 @@ func resourceInstanceRead(ctx context.Context, d *schema.ResourceData, meta inte
 		d.Set("default_association_route_table_id", utils.PathSearch("instance.default_association_route_table_id", getInstanceRespBody, nil)),
 		d.Set("availability_zones", utils.PathSearch("instance.availability_zone_ids", getInstanceRespBody, nil)),
 		d.Set("auto_accept_shared_attachments", utils.PathSearch("instance.auto_accept_shared_attachments", getInstanceRespBody, nil)),
+		d.Set("tags", utils.FlattenTagsToMap(utils.PathSearch("instance.tags", getInstanceRespBody, nil))),
 	)
 
 	return diag.FromErr(mErr.ErrorOrNil())
 }
 
+func deleteResourceTags(client *golangsdk.ServiceClient, resourceType, resourceId, key string) error {
+	var (
+		deleteTagsHttpUrl = "v3/{project_id}/{resource_type}/{resource_id}/tags/{key}"
+	)
+
+	deleteTagsPath := client.Endpoint + deleteTagsHttpUrl
+	deleteTagsPath = strings.ReplaceAll(deleteTagsPath, "{project_id}", client.ProjectID)
+	deleteTagsPath = strings.ReplaceAll(deleteTagsPath, "{resource_type}", resourceType)
+	deleteTagsPath = strings.ReplaceAll(deleteTagsPath, "{resource_id}", resourceId)
+	deleteTagsPath = strings.ReplaceAll(deleteTagsPath, "{key}", key)
+
+	deleteTagsOpt := golangsdk.RequestOpts{
+		KeepResponseBody: true,
+		OkCodes: []int{
+			204,
+		},
```

Comment:
For delete method, do not need write `OkCodes` 204

Reply:
sure and fixed.